### ISSUE TITLE
[Extensions] Missing Design Name While Loading Design

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#11691]
### Current Behavior
<img width="325" alt="Screenshot 2024-08-18 at 7 21 07 AM" src="https://github.com/user-attachments/assets/60306373-ca4e-4a11-9548-121e9cda4d66">

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Meshery Build and Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 UI Design For Meshery Extensions [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 📚 [Meshery Extensibility Providers](https://docs.meshery.io/extensibility/providers)
- ⌨️ [Meshery API Docs](https://docs.meshery.io/extensibility/api)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by leecalcote** [comment]
@amitamrutiya would you like to tackle this one?

**Comment by amitamrutiya** [comment]
Yes @leecalcote. I have a few other issues assigned to me for Meshery UI. Once I finish those, I'll immediately start working on this.

**Comment by leecalcote** [comment]
@amitamrutiya, thanks for volunteering. Given that your plates overflows, I'll reassign.
